### DISCUSSION TITLE
https://plugin-search-pro.vuejs.press/guide.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#670] [Announcements]
# https://plugin-search-pro.vuejs.press/guide.html

vuepress-plugin-search-pro is a powerful client-side search plugin that can support custom indexing and full-text search. Get Started Index Range By default, the plugin will onl...

https://plugin-search-pro.vuejs.press/guide.html

**Comment by joseaquino5** [comment]
"Since searching is done in a Web Worker, setting options to function-typed value is not supported."

How can I use the filter function to filter on the obtained records ?

**Reply by joseaquino5** [reply]
@Mister-Hope any inputs on this ?

**Reply by Mister-Hope** [reply]
Maybe a hook called handleResults 

Open a new discussion would be nice

**Reply by joseaquino5** [reply]
I don't think handleResults can help me per docs specifications (https://plugin-search-pro.vuejs.press/guide.html#customize-search-options): 

![image](https://github.com/vuepress-theme-hope/giscus-discussions/assets/17714907/633da1a4-be51-499c-8eac-cf815f311c3c)

I will open the discussion, thanks!

**Reply by joseaquino5** [reply]
@Mister-Hope I created this https://github.com/vuepress-theme-hope/giscus-discussions/discussions/673

**Reply by Mister-Hope** [reply]
We add 2 new options now
